### PR TITLE
Document RUST_SRC_REPO environment variable.

### PR DESCRIPTION
**Pull request by anp**

### CODE REVIEW

**Comment by hellow554** [comment]
There seems to be no progress here? 

ping @anp

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This tool bisects either Rust nightlies or CI artifacts.
 
+## Setup
+
+Optionally, set the `RUST_SRC_REPO` environment variable to point at your local copy of https://github.com/rust-lang/rust. If the variable is not defined, `cargo-bisect-rustc` will clone it for you which takes a while.
```
Comment (Nashenas88) [inline README.md]: One detail missing here is that this has to be set at compile time. It's imported with the `option_env!` macro. See [here](https://doc.rust-lang.org/std/macro.option_env.html)

**Comment by anp** [comment]
Hi! I obviously haven’t had time to pick this back up, am happy to fix it up for merge if the amount here is valuable.

**Comment by spastorino** [comment]
Hey @anp, I think we you're doing is valuable if you can pick this up pretty sure we can merge it. Unsure what you plan on adding exactly but remember that there's the [Tutorial](https://github.com/rust-lang-nursery/cargo-bisect-rustc/blob/master/TUTORIAL.md).

**Comment by spastorino** [comment]
I think we should land whetever is good enough for now and if you later find more info you can open a new PR.

**Comment by spastorino** [comment]
Hey, this information is very valuable but have you seen that there's some explanation for that variable already on the TUTORIAL.md file. Sorry for making you spend time on this, I'm just jumping here to help. I've just figured that your PR has more than 1 year, probably the TUTORIAL.md file was added later, unsure. Take a look if there's some way that you can fill your information and make it pair nicely with the TUTORIAL or if everything it's already said there we can just close this I guess.

**Comment by anp** [comment]
Let’s give this a close then, no worries!

**Pull request closed without merging**